apps/language_server/lib/language_server/providers/execute_command/manipulate_pipes.ex

```diff
@@ -70,13 +71,24 @@ defmodule ElixirLS.LanguageServer.Providers.ExecuteCommand.ManipulatePipes do
             {:ok, function_call, call_range} =
               get_function_call(line, col, acc.walked_text, current_char, remaining_text)
 
-            {remaining_text,
-             %{
-               acc
-               | walked_text: acc.walked_text <> current_char,
-                 function_call: function_call,
-                 range: call_range
-             }}
+            if function_call_includes_cursor(call_range, line, col) do
+              {remaining_text,
+               %{
+                 acc
+                 | walked_text: acc.walked_text <> current_char,
+                   function_call: function_call,
+                   range: call_range
+               }}
+            else
+              # The cursor was not inside a function call so we cannot
+              # manipulate the pipes
+              {remaining_text,
+               %{
+                 acc
+                 | walked_text: acc.walked_text <> current_char,
+                   function_call: nil
```

Comment:
```suggestion
                 | walked_text: acc.walked_text <> current_char
```

Is this `function_call: nil` needed?

Reply:
No it's not since it was already `nil` in the initial value. I just explicitly set it as `nil` to be more explicit.